.github/workflows/scripts/coverage.sh

```diff
@@ -0,0 +1,44 @@
+!#/usr/bin/env bash
+
+# Ensure past coverage files are deleted (useful for running locally)
+echo "Removing stale coverage files..."
+find . -name "*.profraw" -type f -delete
+find . -name "*.profdata" -type f -delete
+rm -r coverage
+mkdir coverage
+
+# Switch to nightly toolchain, in case running locally
+rustup override set nightly
```

Comment:
Maybe run cargo specifying nightly instead of changing the users default toolchain.
`cargo +nightly ...`

Reply:
Addressed in latest commit.